src/components/SearchBar.tsx

```diff
@@ -2,7 +2,7 @@ import React, { useState } from 'react';
 
 import { Button } from './UI/Button';
 import { Input } from './UI/Input';
-import { NetworkSelector } from './NetworkSelector';
+// import { NetworkSelector } from './NetworkSelector';
```

Comment:
@eyalben let's keep the network selector in, we've added the support on the backend